### PR TITLE
Made EndPoint model, serializer and the fixture

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: data_ocean/fixtures/register.json [hunk 1]
```diff
@@ -8,13 +8,23 @@
...
       "source_name": "pep.org.ua",
       "source_register_id": null,
-      "list": "/api/pep/",
       "retrieve": "/api/pep/{id}/",
       "url_address": "https://pep.org.ua/uk/data",
       "api_address": "https://pep.org.ua/opendata/persons/json",
       "source_last_update": "2020-09-07"
     }
   },
+  {
```
Comment (IegorSoboliev) [inline data_ocean/fixtures/register.json:17]: Is that a good idea to put it here?

Comment (roman-tiukh) [reply data_ocean/fixtures/register.json:17]: its ok.

Comment (roman-tiukh) [reply data_ocean/fixtures/register.json:17]: 1. Do it with all registers and all endpoints.
2. Then remove the list and retrieve fields from the Register model.
3. I think that the name_eng field is also needed.

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: data_ocean/fixtures/register.json (hunk 1 above)
Comment (IegorSoboliev) [reply data_ocean/fixtures/register.json:16]: done

**Review by PyDev777 — APPROVED** [review]

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**